CONTRIBUTING.md

```diff
@@ -12,6 +12,20 @@ You can contribute in the following ways:
 
 If your contributions are accepted into Mastodon, you can request to be paid through [our OpenCollective](https://opencollective.com/mastodon).
 
+## New to Open Source? START HERE
+
+We want all members of the GitHub community to feel welcome when it comes to contributing to our project. You can follow these step-by-step instructions to make a contribution via a pull request.
+
+1. Fork a repository from Mastodon and create a branch to build new features or test out ideas - instructions available here: https://help.github.com/en/articles/fork-a-repo
```

Comment:
These instructions are really long and there's a much easier way to make pull requests by using the Github web editor for small changes.